examples/with-sass/app/routes/index.tsx

```diff
@@ -0,0 +1,36 @@
+export default function Index() {
+  return (
+    <div>
+      <h1>Welcome to Remix</h1>
+      <ul>
+        <li>
+          <a
+            target="_blank"
+            href="https://remix.run/tutorials/blog"
+            rel="noreferrer noopener"
+          >
+            15m Quickstart Blog Tutorial
+          </a>
+        </li>
+        <li>
+          <a
+            target="_blank"
+            href="https://remix.run/tutorials/jokes"
+            rel="noreferrer noopener"
+          >
+            Deep Dive Jokes App Tutorial
+          </a>
+        </li>
+        <li>
+          <a
+            target="_blank"
+            href="https://remix.run/docs"
+            rel="noreferrer noopener"
+          >
+            Remix Docs
+          </a>
+        </li>
+      </ul>
+    </div>
```

Comment:
Could you remove the irrelevant stuff that can be removed (like this bit) and comment the primary relevant bits?